src/Compilers/Core/Portable/PEWriter/BlobWriter.cs

```diff
@@ -321,14 +272,14 @@ internal void WriteDecimal(decimal value)
             value.GetBits(out isNegative, out scale, out low, out mid, out high);
 
             WriteByte((byte)(scale | (isNegative ? 0x80 : 0x00)));
-            WriteUint(low);
-            WriteUint(mid);
-            WriteUint(high);
+            WriteUInt32(low);
+            WriteUInt32(mid);
+            WriteUInt32(high);
         }
 
-        internal void WriteDateTime(DateTime value)
+        public void WriteDateTime(DateTime value)
         {
-            WriteLong(value.Ticks);
+            WriteInt64(value.Ticks);
```

Comment:
I believe this should be ```value.ToBinary()``` (to adjust for time-zones)
However, is ```WriteDateTime``` actually used?